docs/How-We-Work/Branching-Strategy.md

```diff
@@ -0,0 +1,20 @@
+## Dev Workflow and Branching Strategy
+
+Moving forward, the dev team will be shifting our branching strategy to help alleviate some of our dependency issues. It won't solve the problem completely, but it should make the working environment considerably easier to manage for all of us.
+
+### Part I
+
+Frontend and backend work will start working on separate issues
+* This is a major source of our issues as the frontend frequently has to pull in backend code to build the frontend off of. However, the frontend has the option of creating mock data so it isn't initially dependent on the backend. So, if the backend is working on one issue while the frontend works on a completely different issue, both can move forward without holding each other up.
+* Once either the frontend or backend finishes an issue we will check to see if the other end has an issue finished. If so, that will be the issue the developer works on next, so we can still push through working full stack code at a good pace.
```

Comment:
I love this part — let FE and BE work independently, but focus on completing whole slices of user-facing functionality and don't let one side get too far ahead of the other.